django/db/models/expressions.py

```diff
@@ -1132,6 +1132,12 @@ def get_group_by_cols(self):
 
     def reverse_ordering(self):
         self.descending = not self.descending
+        if self.nulls_last:
```

Comment:
How about:
```
if self.nulls_first or self.nulls_last:
    self.nulls_first = not self.nulls_first
    self.nulls_last = not self.nulls_last
```

Reply:
OK

`self.nulls_first, self.nulls_last = self.nulls_last, self.nulls_first` is also tempting..